syntax = "proto3";

package election_timeout;

message ElectionTimeoutRequest {
    uint32 timeout = 1;
}
